### ISSUE TITLE
construction spray/cans

### ISSUE THREAD

**Issue by bear989** [Blood-Asp/GT5-Unofficial#1190]
I could swear they existed before in GT5 but havent been able to track them down on multiple GT5u releases. but there should/was a recipe to create construction foam in a mixer that then could be caned in a fluid caner to make the GT cf spray cans, there also used to be a hardener witch was just canned sand, and a remover spray. it is ridiculous to have to make a IC2 fluid caner just for CF or making the reinforced stone for reactor rooms.

**Comment by MauveCloud** [comment]
idk about a spray can of construction foam, but in 5.08 at least, it's possible to put a steel frame box in a chemical bath with liquid concrete to get reinforced stone, so you don't even need construction foam for that.

**Comment by OvermindDL1** [comment]
Yeah I remember what @bear989 says too, it should be there...  Maybe it was GT4?  But I definitely remember it too.

**Comment by leagris** [comment]
#### Hydration Spray:
- Re-add hydration spray for crops.
- Also used as undying spray.

#### About Dyes Sprays:
While you are at spray cans. Please re-enable functional dye sprays as in GT4:
- Make it way more economical than using raw dyes with 64 uses per spray can.
- Allow dying in-world
- Allow as multi-use dye recipe ingredient in crafting-grid.

#### If possible:
- Allow dying cables, pipes and machines before in-world placement.
- Allow removed cables and pipes to retain their dye

#### Greatly missed GT5-GT4 regression:
- Make cables and pipes compatible with IC2 foam and Obscurator texturing.
- Fix tileMachine rendering with ambient occlusion. (smooth lighting).

#### Optional spray addition:

##### Enable arbitrary fluid to spray
- May not allow hot fluids
- Each use of spray (except hydration) contributes to chunk pollution.
- Potions spray (short range multi-use)
- Chlorine spray (undying)
- Glue spray as workbench alternative to some assembler recipes.
- Fuel spray as flame thrower weapon crafting ingredient.
- Milk cream spray for food
- Epoxy spray on any block. (block proxy tile to render block texture with saturated hue for varnished effect)
- Polyethylene spray add a polyethylene sheet on block face.
- Polyurethane foam spray on machine (sound deadener) 
-

**Comment by Antillar** [comment]
@Blood-Asp this to https://github.com/Blood-Asp/GT5-Unofficial/issues/654 maybe?
The original addition request is fulfilled.
